docs/architecture/microservices/toc.yml

```diff
@@ -60,12 +60,12 @@
       href: architect-microservice-container-applications/resilient-high-availability-microservices.md
     - name: Orchestrating microservices and multi-container applications for high scalability and availability
       href: architect-microservice-container-applications/scalable-available-multi-container-microservice-applications.md
-  - name: Development Process for Docker Based Applications
+  - name: Development Process for Docker-Based Apps
     href: docker-application-development-process/index.md
     items:
     - name: Development workflow for Docker apps
       href: docker-application-development-process/docker-app-development-workflow.md
-  - name: Designing and Developing Multi Container and Microservice Based NET Applications
+  - name: Design and Develop Multi-Container and Microservice-Based .NET Apps
```

Comment:
This title is almost identical to the earlier "Architecting..." one.  What's the actual difference between the two / what's the difference between designing and "architecting"?

Reply:
I'm not the original author of this TOC, but I think the key difference is the multi-container aspect.